index.js

```diff
@@ -1,7 +1,63 @@
+const worksData = [
+  {
+    id: 1,
+    title: "Project 1",
+    image: "./assets/images/work-img-1.png",
+    description:
+      "1 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
```

Comment:
- [ ] the objects should contain technologies and a link to the live version as well.